src/main/resources/application.yaml

```diff
@@ -61,6 +61,8 @@ dm:
 
 scheduling:
   enabled: ${SCHEDULING_ENABLED:false}
+  pool: ${SCHEDULING_POOL:10}
+  lockAtMostFor: ${SCHEDULING_LOCKATMOSTFOR:PT10M} # 10 minutes
```

Comment:
it would be good to add a comment on what the format here is (unless sample `PT10M` is clear to everyone...) imho

Reply:
how about `LOCK_AT_MOST_FOR` instead of `LOCKATMOSTFOR`?

Reply:
I don't mind either way. Taken that approach after reading `org.springframework.core.env.SystemEnvironmentPropertySource#checkPropertyName` implementation so that `scheduling.lockAtMostFor` gets resolved to `SCHEDULING_LOCKATMOSTFOR`.

True that, perhaps too defensive that ended up too obscure. Adding underscores.